bundle_pt.properties

```
@@ -217,29 +217,29 @@ are_you_sure_you_wish_to_leave_the_party_question=Tem certeza de que deseja sair
 invite=Convidar
 inviting_dots=Convidando...
 leave=Sair
-you_arent_invited_to_that_party=Você não foi convidade para esse grupo :(
+you_arent_invited_to_that_party=Você não foi convidado para esse grupo :(
 that_party_doesnt_exist=Esse grupo não existe :(
 you_are_already_in_a_party=Você já está em um grupo:(
 unable_to_join_party=Não é possível entrar no grupo :(
-party_is_full=O grupo tá cheio :(
+party_is_full=O grupo está cheio :(
 that_player_is_not_online=Esse jogador não está online :(
 they_are_already_in_a_party=Eles já estão em um grupo :(
-they_are_already_invited=Eles já estão convidados :P
-too_many_open_invites=Muitos convites abertos :(
+they_are_already_invited=Eles já foram convidados :P
+too_many_open_invites=Você tem muitos convites em aberto :(
 
 # Character Details
 
 trade=Negociar
 whisper=Sussurrar
-invite_to_party=Convidar ao Grupo
-revoke_invite=Revogar Convite
+invite_to_party=Convidar pro Grupo
```

This is grammatically incorrect. The former is the better
```suggestion
invite_to_party=Convidar ao Grupo
```